test/NuGet.Core.Tests/NuGet.Protocol.Tests/LocalFolderUtilityTests.cs

```diff
@@ -1030,6 +1032,65 @@ public void LocalFolderUtility_GetPackagesByIdV3NotFoundMissingDir()
             }
         }
 
+        [Fact]
+        public void LocalFolderUtility_EnsurePackageFileExists_ThrowsWhenEmpty()
+        {
+            //Arrange
+            List<string> testList = new List<string>();
+            string packagePath = string.Empty;
+
+            //Act
+            try
+            {
+                LocalFolderUtility.EnsurePackageFileExists(packagePath, testList);
+                Assert.True(false, "Did not throw an exception as expected.");
```

Comment:
Isn't this line of code is unreachable if expected behavior is to throw exception ?. May be use `Assert.Throws' directly instead of assert in the exception handler.

Reply:
Correct. The line is designed to fail if the code doesn't throw an exception. I didn't know about Assert.Throws and can take a look at that since it sounds cleaner.

Reply:
This look good, now ?